crowdsourcing/serializers/task.py

```diff
@@ -1,14 +1,36 @@
-__author__ = 'elsabakiu, asmita, megha'
+__author__ = 'elsabakiu, asmita, megha,kajal'
 
 from crowdsourcing import models
 from rest_framework import serializers
 
 class TaskSerializer(serializers.ModelSerializer):
-    class Meta:
-        model = models.Task
-        fields = ('module', 'statuses', 'status', 'deleted', 'created_timestamp', 'last_updated')
-        read_only_fields = ('module', 'statuses', 'created_timestamp', 'last_updated')
+	deleted = serializers.BooleanField(read_only=True)
```

Comment:
I have used tab size 4(indentation). In my settings too of sublime-text i have the same and here i clearly see the error. I didn't know how that occurred or how to fix it.
what is the tab size we are using for project? seems 2.